### ISSUE TITLE
No window in wayland

### ISSUE THREAD

**Issue by lholden** [tomaka/glutin#849]
Running any of the examples under wayland doesn't seem to be presenting a window. I've tried both glutin 0.7.2 and from the master branch.

Fedora 25, Gnome 3.22.2 (Wayland/Mutter), Intel HD Graphics 530

**Comment by tomaka** [comment]
This is probably a duplicate of https://github.com/tomaka/winit/issues/109#issuecomment-274185472

A workaround to the problem is to draw and swap buffers once before starting to poll/wait events.
It's unclear what the proper way to fix it is.
